README.md

```diff
@@ -1,11 +1,13 @@
-1. Open Command line.
-2. Go to the folder with the cloned project.
-3a. Enter command: javac CommandLineParams1.java (for compiling).
-if ok:
-4a. Enter command with three additional symbols: java CommandLineParams1 2 4 6(run programm).
-3b. Enter command: javac CommandLineName.java (for compiling).
-if ok:
-4b. Enter command with Name: java CommandLineName [any name](run programm).
-3c. Enter command: javac CommandLineCalc.java (for compiling).
-if ok:
-4c. Enter command with two additional numbers: java CommandLineCalc 3 6(run programm).
\ No newline at end of file
+# Homework readme
+ **General rules**
+1.  Open Command line.
+2.  Go to the folder with the cloned project. 
+### 2.3.3 
+1. Enter command: 
+```
+javac HtmlTable2.java
+```
+2. If compiling java file was successful enter command:
+```
+java HtmlTable2 [path to your file/directory]
```

Comment:
It would be better to print help info by follow command
```
java HtmlTable2 --help
```
Example:
```
usage: HtmlTable2 [-p | --path ] path to your directory only
```